CardanoSharp.Wallet/Extensions/StringExtension.cs

```diff
@@ -2,21 +2,26 @@
 
 namespace CardanoSharp.Wallet.Extensions
 {
-    public static class StringExtension
-    {
-        public static string ToStringHex(this byte[] bytes)
-        {
-            var hex = BitConverter
-                .ToString(bytes)
-                .Replace("-", "")
-                .ToLower();
+	public static class StringExtension
+	{
+		public static string ToStringHex(this byte[] bytes)
+		{
+			var hex = BitConverter
+				.ToString(bytes)
+				.Replace("-", "")
+				.ToLower();
 
-            return hex;
-        }
+			return hex;
+		}
 
-        public static byte[] ToBytes(this string value)
-        {
-            return System.Text.Encoding.UTF8.GetBytes(value);
-        }
-    }
-}
+		public static string ToStringAscii(this byte[] bytes)
+		{
+			return System.Text.Encoding.UTF8.GetString(bytes);
+		}
```

Comment:
This seems weird... method is for ASCII but returns UTF8 which is exactly the method below

Reply:
ahh sorry, yeah thats a copy paste mistake, that shouldn't have been in there

Reply:
added it back as i think it is useful, but changed the name from ToStringAscii to GetString